lms/services/vitalsource/service.py

```diff
@@ -215,7 +215,10 @@ def check_h_license(
             # Do the actual license check, only for students
             user_reference = self.get_user_reference(lti_params)
             assert self._sso_client
-            if not self._sso_client.get_user_book_license(user_reference, self.H_SKU):
+            for sku in self.H_SKUS:
+                if self._sso_client.get_user_book_license(user_reference, sku):
+                    return None
+            else:
```

Comment:
[else of the for loop](https://docs.python.org/3/tutorial/controlflow.html#else-clauses-on-loops)

If we went through all skus, return the no license code.